src/parser.js

```diff
@@ -3,11 +3,17 @@
 const handle = require('./handle.js');
 const templatizer = require('./templatizer.js');
 const validate = require('./configValidation.js');
+const configDefault = require('./template/configDefault.json');
+const noDocsError = new Error('No documentation passed');
 
-const parser = (help, config) => {
-    config = validate(config);
-    const context = handle(help, config);
```

Comment:
`Handle` is too generic and without docs gives no idea what it handles.

Reply:
@xferra done